cmd/erasure-server-pool-decom.go

```diff
@@ -881,6 +886,42 @@ func (z *erasureServerPools) decommissionPool(ctx context.Context, idx int, pool
 	return nil
 }
 
+//msgp:ignore decomMetrics
+type decomMetrics struct{}
+
+var globalDecommissionMetrics decomMetrics
+
+//msgp:ignore decomMetric
+//go:generate stringer -type=decomMetric -trimprefix=decomMetric $GOFILE
+type decomMetric uint8
+
+const (
+	decomMetricDecommissionBucket decomMetric = iota
+	decomMetricDecommissionObject
+	decomMetricDecommissionRemoveObject
+)
+
+func decomTrace(d decomMetric, poolIdx int, startTime time.Time, duration time.Duration, path string) madmin.TraceInfo {
+	return madmin.TraceInfo{
```

Comment:
@krisis there is a field in madmin.TraceInfo{} which is called Message. It doesn't seem to be used right now in minio/minio and minio/mc. I think we can break it and call it Error and fill it when decomissioning returned an error.

Reply:
@vadmeste That's a good idea. I have made the necessary changes in madmin-go and mc too. 
Sample error trace from `mc admin trace --call decommission ...`,
```
2022-08-09T11:11:34.442 [DECOMMISSION] decommission.DecommissionBucket (pool-id=0) minio1:9000 .minio.sys err='this is a test error' 50.13228ms
```